README.md

```diff
@@ -65,10 +66,10 @@ Define your xml as:
         android:id="@+id/shimmer_recycler_view"
         android:layout_width="match_parent"
         android:layout_height="match_parent"
-        app:demo_child_count="10"
-        app:demo_grid_child_count="2"
-        app:demo_layout="@layout/layout_demo_grid"
-        app:demo_layout_manager_type="grid"
+        app:shimmer_demo_child_count="10"
```

Comment:
Great job that you have noticed the readme was not updated! Could please add it to a different commit because it is not related to the maskWidth feature?